$NetBSD: patch-gr-audio_lib_CMakeLists.txt,v 1.4 2023/01/23 19:26:20 adam Exp $

If alsa is present on the system but the PKG_OPTION disabled we
should not use it.

--- gr-audio/lib/CMakeLists.txt.orig	2022-12-19 14:34:48.000000000 +0000
+++ gr-audio/lib/CMakeLists.txt
@@ -21,7 +21,7 @@ list(APPEND gr_audio_confs ${CMAKE_CURRE
 ########################################################################
 ## ALSA Support
 ########################################################################
-find_package(ALSA)
+@@ALSA@@find_package(ALSA)
 
 if((ALSA_FOUND)
    AND NOT (CMAKE_SYSTEM_NAME STREQUAL "kFreeBSD")
